shared/chat/inbox/index.desktop.js

```diff
@@ -170,8 +171,8 @@ const _containerStyle = {
   backgroundColor: globalColors.blueGrey,
   contain: 'strict',
   height: '100%',
-  maxWidth: 260,
-  minWidth: 260,
+  maxWidth: inboxWidth,
```

Comment:
is there a reason why we're setting min and max width as opposed to just width?

Reply:
I'm not sure, I didn't want to modify these styles, just to factor `260` out into a constant